packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Dialog/index.tsx

```diff
@@ -162,6 +162,18 @@ export const Dialog: React.FC = () => {
                         <Menu.Item onSelect={() => setEditing(true)}>
                           Edit Comment
                         </Menu.Item>
+                        <Menu.Item
+                          onSelect={() => {
+                            effects.browser.copyToClipboard(
```

Comment:
Okay, I have a suggestion here.

You can create an action in `comments` called like: `copyPermalinkToClipboard`, where you pass the `commentId`. Inside that action you do what you do here, run the two effects.

The great thing now is that both `Dialog` and `Comment` will just call this action and we avoid duplicate logic 😄